### DISCUSSION TITLE
LaTeX Logo in presentation revealjs

### DISCUSSION THREAD

**Discussion by MarcosDelgadoAlcantar** [quarto-dev/quarto-cli#5633] [Q&A]
I'm making a presentation with revealjs and it's about LaTex. I tried to use \LaTex{} to put the LaTeX logo. It didn't work. I used plastex to convert a TEX file to HTML format so the logo looks like L<sup style="font-variant:small-caps; margin-left:-0.3em">a</sup>T<sub style=" text-transform:uppercase;margin-left:-0.2em">e</sub>X
It works, but I would like to know if there is an option more linked to quarto and revealjs to put it in my work.

**Comment by cderv** [comment]
You can probably use this extension: https://github.com/quarto-ext/fancy-text

**Reply by MarcosDelgadoAlcantar** [reply]
Thank you very much, it works!!!
The only detail is that you have to install the extension in the same directory where the presentation you are creating is located.

**Reply by cderv** [reply]
> The only detail is that you have to install the extension in the same directory where the presentation you are creating is located.

Yes this is how extension works. Same folder as the document, or at project level otherwise
